dnsviz/commands/grok.py

```diff
@@ -248,6 +248,10 @@ def build_parser(self, prog):
                 const=True, default=False,
                 action='store_const',
                 help='Trust all CDNSKEY/CDS records')
+        self.parser.add_argument('--allow-multisigner',
+                const=True, default=False,
+                action='store_const',
+                help='Don\'t issue errors for missing DNSKEYs in multi-signer setups')
```

Comment:
> I think the description may indicate that DNSViz will tolerate any kind of a missing DNSKEY.
> 

Yeah, it's always tricky to try to make it both descriptive and terse.  I don't love the part about "... as long as the DNSKEYs are validated correctly" because it actually doesn't check that.  For this particular error and exception, it only checks if the key 1) does not sign the ZSK, 2) corresponds to a DS, and 3) (just added) signs the DNSKEY RRset (i.e., is a KSK).  The last addition was an oversight before.

So based on that, here is my suggestion:

"Don't issue errors for missing KSKs with DS RRs (e.g., multi-signer setups, RFC 8901)"

Thoughts?

Reply:
I know that the DNSKEYs (with missing KSK) aren't checked in this place but aren't all individual responses from all servers validated? If so then I think that the valid path is basically ensured.

I like your suggestion!